test/modules/CloudToDeviceMessageTester/CloudToDeviceMessageReceiver.cs

```diff
@@ -46,14 +64,18 @@ public async Task ReportTestResult(Message message)
             await ModuleUtil.ReportTestResultAsync(this.testResultReportingClient, this.logger, testResultReceived);
         }
 
-        public override async Task StartAsync(CancellationToken ct)
+        public async Task StartAsync(CancellationToken ct)
```

Comment:
change all public methods access modifier to internal if possible (can't change for Dipsose() method).